lib/zold/upgrades.rb

```diff
@@ -10,11 +10,18 @@ module Zold
   # If the data is up-to-date, the version of the data is equal to
   # Zold::VERSION.
   #
-  # If the version is lower than Zold::VERSION, upgrade scripts from
-  # `upgrades/` have to run. They are named `<version>.rb`, so for
-  # instance `upgrades/0.0.1.rb` etc.
+  # By comparing Zold::VERSION with the data version we determine
+  # which upgrade scripts will be executed.
   #
-  # Only the scripts from the data version up need to run.
+  # If the data version is the same as Zold::VERSION, the data is
+  # up to date.
+  #
+  # If the data version is lower than Zold::VERSION, we will look into
+  # the `upgrades/` directory and any scripts from the data version
```

Comment:
@botanicus please don't use `data version up`, it's ambiguous at least to me. You can also ask @yegor256 to resolve our conflict.